cmd/signature-v4-postpolicyform_test.go

```diff
@@ -33,8 +34,6 @@ func TestPostPolicyForm(t *testing.T) {
 		ErrCode       APIErrorCode
 	}
 	testCases := []testCase{
-		// Mal formed policy document
-		{Bucket: "testbucket", Key: "user/user1/filename/${filename}/myfile.txt", XAmzDate: "20160727T000000Z", XAmzAlgorithm: "AWS4-HMAC-SHA256", ContentType: "image/jpeg", Policy: "==", ErrCode: ErrMalformedPOSTRequest},
 		// Different AMZ date
```

Comment:
it's part of the standard library, so we need not test it, note that base64.StdEncoding.DecodeString is not used inside checkPostPolicy anymore